ast.go

```diff
@@ -246,47 +230,65 @@ func (node *Subquery) ContainsSubquery() bool {
 	return true
 }
 
-// ParenTableExpr represents a parenthesized list of TableExpr.
+// ParenTableExpr represents a parenthesized TableExpr.
 type ParenTableExpr struct {
-	TableExprList TableExprList
+	TableExpr TableExpr
 }
 
 // String returns the string representation of the node.
 func (node *ParenTableExpr) String() string {
-	return fmt.Sprintf("(%v)", node.TableExprList.String())
+	return fmt.Sprintf("(%v)", node.TableExpr.String())
+}
+
+// JoinOperator represents a join operator.
+type JoinOperator struct {
+	Op      string
```

Comment:
[Here](https://github.com/tablelandnetwork/go-sqlparser/pull/33/files#diff-0945956cd950d047e7fa866dcf34fee7d29cbce277d4e5e5cd621f2213f3be6cR273-R278) is the list of possible join operations

Reply:
The CROSS JOIN operation is represented as a simple JOIN since a CROSS JOIN is a JOIN without an ON. From [docs](https://www.sqlite.org/lang_select.html#determination_of_input_data_from_clause_processing_):

> If the join-operator is "CROSS JOIN", "INNER JOIN", "JOIN" or a comma (",") and there is no ON or USING clause, then the result of the join is simply the cartesian product of the left and right-hand datasets.